### PR TITLE
[Feature]Support show create external catalog

### LINKED ISSUE #16508: Support show create external catalog
## Feature request

**Is your feature request related to a problem? Please describe.**

Support show create external catalog

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by leoyy0316**
## What type of PR is this：
- [ ] BugFix
- [x] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #16508

## Problem Summary(Required) ：

support show create external catalog
## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Comment by leoyy0316** [comment]
run starrocks_fe_unittest

**Comment by leoyy0316** [comment]
run starrocks_fe_unittest

File: fe/fe-core/src/main/java/com/starrocks/sql/ast/ShowCreateExternalCatalogStmt.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import com.starrocks.catalog.Column;
+import com.starrocks.catalog.ScalarType;
+import com.starrocks.qe.ShowResultSetMetaData;
+
+// SHOW CREATE EXTERNAL CATALOG statement.
+public class ShowCreateExternalCatalogStmt extends ShowStmt {
+    private static final ShowResultSetMetaData META_DATA =
+            ShowResultSetMetaData.builder()
+                    .addColumn(new Column("catalog", ScalarType.createVarchar(20)))
```
Comment (stephen-shelby) [inline fe/fe-core/src/main/java/com/starrocks/sql/ast/ShowCreateExternalCatalogStmt.java:26]: "Catalog"

File: fe/fe-core/src/main/java/com/starrocks/sql/ast/ShowCreateExternalCatalogStmt.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                    .addColumn(new Column("Create External Catalog", ScalarType.createVarchar(30)))
```
Comment (stephen-shelby) [inline fe/fe-core/src/main/java/com/starrocks/sql/ast/ShowCreateExternalCatalogStmt.java:27]: Create Catalog

**Review by stephen-shelby — DISMISSED** [review]

**Comment by leoyy0316** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 30 / 41 (73.17%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|0|3|00.00%|[1361, 1362, 1363]|
|:large_blue_circle:|com/starrocks/sql/ast/AstVisitor.java|0|1|00.00%|[283]|
|:large_blue_circle:|com/starrocks/sql/analyzer/ShowStmtAnalyzer.java|0|4|00.00%|[673, 674, 675, 677]|
|:large_blue_circle:|com/starrocks/sql/ast/ShowCreateExternalCatalogStmt.java|10|11|90.91%|[47]|
|:large_blue_circle:|com/starrocks/qe/ShowExecutor.java|20|22|90.91%|[1981, 1982]|

**Review by stephen-shelby — APPROVED** [review]

**Review by Youngwb — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Pull request merged**